src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TodosSortType } from '../../types/TodosSortType';
+
+type Props = {
+  todosSortBy: TodosSortType,
+  onSortByField: (sortBy: TodosSortType) => void,
+  completedTodosLength: number,
+  clearAllCompleted: () => void,
+  todosLength: number,
+};
+
+export const Footer: React.FC<Props> = ({
+  onSortByField,
+  todosSortBy,
+  completedTodosLength,
+  clearAllCompleted,
+  todosLength,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todosSortBy === TodosSortType.All
+          ? `${todosLength - completedTodosLength} items left`
+          : `${todosLength} items left`}
```

Comment:
It is better to put the number calculation in a separate variable for better readability.

Reply:
> It is better to put the number calculation in a separate variable for better readability.

Don't forget about all comments